### PR TITLE
fix warnings

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

File: src/fragmentarium/ui/images/CdliImages.test.tsx [hunk 1]
```diff
@@ -21,8 +26,9 @@ beforeEach(async () => {
 })
 
 describe('CdliImages', () => {
-  beforeEach(() => {
+  beforeEach(async () => {
     renderImages()
+    await waitForElementToBeRemoved(() => screen.getByLabelText('Spinner'))
```
Comment (jlaasonen) [inline src/fragmentarium/ui/images/CdliImages.test.tsx:31]: Spinners are used a lot. There could be `waitForSpinners` function.

File: src/common/ErrorBoundary.test.tsx [hunk 2]
```diff
@@ -31,19 +34,24 @@ describe('Children throw an error', () => {
...
   })
 
   it('Clicking report button opens report dialog', async () => {
     clickNth(element, 'Send a report', 0)
 
     expect(errorReportingService.showReportDialog).toHaveBeenCalled()
   })
+  afterEach(async () => {
+    spy.mockRestore()
+  })
```
Comment (jlaasonen) [inline src/common/ErrorBoundary.test.tsx:54]: Why do you need to restore manually and multiple times?

File: yarn.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+yarn run v1.22.17
+$ react-scripts test --watchAll=false
+info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
Comment (jlaasonen) [inline yarn.txt:3]: What's this for?

File: src/common/ErrorBoundary.test.tsx [hunk 4]
```diff
@@ -10,8 +10,11 @@ describe('Children throw an error', () => {
   let element
   let error
   let errorReportingService
+  let spy
 
   beforeEach(async () => {
+    spy = jest.spyOn(console, 'error')
+    spy.mockImplementation()
```
Comment (jlaasonen) [inline src/common/ErrorBoundary.test.tsx:17]: Now that `error` is spied, you could also verify it was called or not called correctly.

File: src/fragmentarium/ui/images/CdliImages.test.tsx (hunk 1 above)
Comment (yCobanoglu) [reply src/fragmentarium/ui/images/CdliImages.test.tsx:31]: @jlaasonen i would have to pass in screen and the function wouldn't reduce the code much can i just leave it for now ?

Comment (jlaasonen) [reply src/fragmentarium/ui/images/CdliImages.test.tsx:31]: It is not about reducing code. The same logic is duplicated in multiple places. If the spinner is changed all of them have to be updated. If there is a function to wait for the spinner only it has to be changed.

**Review by jlaasonen — APPROVED** [review]

File: src/fragmentarium/ui/images/CdliImages.test.tsx (hunk 1 above)
Comment (yCobanoglu) [reply src/fragmentarium/ui/images/CdliImages.test.tsx:31]: alright then i will do it

**Pull request merged**